unit_tests/extract_functions.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import os
+from pathlib import Path
+
+
+def main():
+    file_path = Path(os.path.abspath(__file__))
```

Comment:
Couldn't this be replaced by a dedicated function in the script that would leverage `declare -F`?